wikipedia.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="wiki.css">
+    <title>Wikipedia-html></title>
+ 
+    <div>
+        <header class="wrapper">
+            <form class="header" action="" method="get"></form>
+            <a href="images/menu-icon.png"><img src="images/menu-icon.png" id="menu-icon" alt=""></a>
+            <a href="https://en.m.wikipedia.org/wiki/Main_Page"><img src="images/wiki-img.png" id="wiki-img" alt=""></a>
+            <!-- <a href=""><img src="images/search-icon.png" id="search-icon" alt=""></a> -->
+            <input type="text" placeholder="Search wikipedia" id="input">
+    </div>
+
+    </header>
+
+
+
+
+
+
+
+
+    <h1>HTML</h1>
+    <hr>
+
+
+
+
+    language / download / star / lock
+
+
+
+    <hr>
+
+    <div role="note" class="">.htm and ".html" redirect here. For other uses, see
+        <a href="https://en.m.wikipedia.org/wiki/HTM">HTM</a>.</div>
+
+    <div role="note" class="">
+        <div style="display:inline;">For the use of HTML on Wikipedia, see
+            <a href="https://en.m.wikipedia.org/wiki/Help:HTML_in_wikitext">Help:HTML in wikitext</a>.</div>
+    </div>
+
+
+
+    HTML
+    <b>Hypertext Markup Language (HTML)</b>
+    <p>is the <a href="link">standard markup language </a>
+        standar for documents designed to be displayed in a <a href="link"> web browser.</a>
+        It can be assisted by technologies such as <a href="link"> Cascading Style Sheets</a> (CSS) and <a href="link">
+            languages</a> such as <a href="link">JavaScript.</a></p>
+
+
+    <div class="html-logo">
+        <h2>HTML Table</h2>
+        <a href="https://en.m.wikipedia.org/wiki/HTML#/media/File%3AHTML5_logo_and_wordmark.svg"><img
+                src="images/html-logo.png" id="logo" alt="">
```

Comment:
Hi @tijanisylla , 

```suggestion
        <a href="https://en.m.wikipedia.org/wiki/HTML#/media/File%3AHTML5_logo_and_wordmark.svg"><img
                src="images/html-logo.png" id="logo" alt="HTML Logo">
```

:100: